driver/node.go

```diff
@@ -337,7 +348,27 @@ func (d *Driver) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolume
 	d.log.WithField("method", "node_expand_volume").
 		Info("node expand volume called")
 
-	return nil, status.Error(codes.Unimplemented, "")
+	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
+	}
+
+	mounter := mount.New("")
+	devicePath, _, err := mount.GetDeviceNameFromMount(mounter, req.GetVolumePath())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Unable to get device path for %q: %v", req.GetVolumePath(), err)
+	}
+
+	r := resizefs.NewResizeFs(&mount.SafeFormatAndMount{
+		Interface: mounter,
+		Exec:      mount.NewOsExec(),
+	})
+
+	if _, err := r.Resize(devicePath, req.GetVolumePath()); err != nil {
```

Comment:
The [CSI spec distinguishes between online and offline resizing](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerexpandvolume). Do I understand correctly that we support the online variant?

Either way, I believe we should expose our capabilities in `GetPluginCapabilities` by what I think should look like this (assuming we support _online_):

```go
Type: &csi.PluginCapability_VolumeExpansion_{
  VolumeExpansion: &csi.PluginCapability_VolumeExpansion{
    Type: csi.PluginCapability_VolumeExpansion_ONLINE,
  },
}
```